examples/platform_channel/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.example.flutter"
+    package="com.example.platformChannel"
```

Comment:
Nit: I don't know if it's enforced, but package names are usually in all lower-case (https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html) - so either com.example.platformchannel or com.example.platform_channel.